### PR TITLE
Bug 1111471 - [Stingray][Home] Pin new cards at the bottom of deck group...

### PR DESCRIPTION
**Pull request by rexboy7**
...s

### CODE REVIEW

**Comment by try-server-hook** [comment]
[rexboy7 Rex KM Lee (rexboy7)](https://github.com/rexboy7) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a0158695c6b1)

File: tv_apps/tv_shared/js/card_manager.js [hunk 1]
```diff
@@ -371,17 +371,40 @@
...
+            continue;
+          } else if (position !== -1 && card.group !== newCard.group) {
+            // We've exceeded the end of the group.
+            break;
+          } else if (card.group === newCard.group) {
+            // We're still inside the group.
+            position = idx;
+          }
+        }
+        ++position;
```
Comment (dwi2) [inline tv_apps/tv_shared/js/card_manager.js]: nit: What is the reason to put `+` in front of `position`? I'd prefer to put it this way
```js
position += 1;
```

**Comment by try-server-hook** [comment]
[rexboy7 Rex KM Lee (rexboy7)](https://github.com/rexboy7) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9a16ccdde6e1)

**Comment by try-server-hook** [comment]
[rexboy7 Rex KM Lee (rexboy7)](https://github.com/rexboy7) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ba2b63cef603)

**Pull request merged**